scripts/import_assets/import_assets.py

```diff
@@ -0,0 +1,85 @@
+import csv
+import json
+import requests
+import argparse
+import os
+from requests.exceptions import HTTPError
+from urllib.parse import urljoin
+
+DEFAULT_API_URL = 'https://se-api.holmsecurity.com/v1/'
+
+
+def read_data(args):
+    # reads data from a csv_file
```

Comment:
Please rename the `read_data` function to something more more explanatory like `read_and_create_assets`.

Also it would be good to add docstrings to all functions in which you had comments. Like:

```
"""
Reads data from CSV file and posts it as JSON to the API.
"""
```

Reply:
Done!